
<launch>

  <!-- Depth Image to Laser Scan Node -->
  <node name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" args="">
    <remap from="image"       to="/camera/depth/image_raw"/>
    <remap from="camera_info" to="/camera/depth/camera_info"/> 

    <!-- PARAMETERS: -->
    <!-- These are the default values. --> 
    <param name="scan_height"     type="int"    value="5"/> <!-- default: 1 pixel. Number of pixel rows used to generate laser scan. -->
    <param name="scan_time"       type="double" value="0.033"/> <!-- default:0.033, 30 FPS . Time between scans. -->
    <param name="range_min"       type="double" value="0.45"/> <!--default:0.45m. Ranges less than this are considered -Inf. -->
    <param name="range_max"       type="double" value="4.0"/> <!--default: 10m. Ranges less than this are considered +Inf. -->
    <param name="output_frame_id" type="str"    value="laser"/> <!--default: camera_depth_frame. Frame id of the laser scan. -->
	
  </node>
	
</launch>

